docs/reference/api/rtos/Event.md

```diff
@@ -6,7 +6,7 @@ The Event class is thread safe. The `post` and `cancel` APIs are IRQ safe.
 
 ### Event class reference
 
-[![View code](https://www.mbed.com/embed/?type=library)](https://os.mbed.com/docs/v5.6/mbed-os-api-doxy/_event_8h_source.html)
+[![View code](https://www.mbed.com/embed/?type=library)](https://os-doc-builder.test.mbed.com/docs/v5.7/mbed-os-api-doxy/_event_8h_source.html)
```

Comment:
Query: Is there a reason we're pointing to .h file instead of to the Event class (https://os-doc-builder.test.mbed.com/docs/v5.7/mbed-os-api-doxy/classevents_1_1_event.html)?

Reply:
After our in-person conversation, I think you're correct, and we should point to this .h file instead of the class page. We can always update this later when the class page is more complete.